src/app/components/OfflineBanner/hook.ts

```diff
@@ -28,12 +28,14 @@ export const useConsensusFreshness = (network: Network) => {
   }
 }
 
-export const useRuntimeFreshness = (scope: SearchScope): FreshnessInfo => {
+export const useRuntimeFreshness = (scope: SearchScope, polling = false): FreshnessInfo => {
```

Comment:
I think `polling` should be a named parameter

Reply:
Done

Reply:
I was thinking `(scope: SearchScope, queryParams: { polling?: boolean } = {}): FreshnessInfo`

Reply:
Yes, I changed it so before merging.